README.md

```diff
@@ -59,26 +59,26 @@ apiVersion: v1
 metadata:
   name: hoge-pod
```

Comment:
BTW: we need to update this name. `hoge` is for Japanese people 😆

Reply:
Whoa, I didn't know that.